src/components/text-input/index.md.njk

```diff
@@ -86,11 +92,25 @@ Do not set the `inputmode` attribute to `decimal` as it causes some devices to b
 
 Do not use `<input type="number">` unless your user research shows that there’s a need for it. With `<input type="number">` there’s a risk of users accidentally incrementing a number when they’re trying to do something else - for example, scroll up or down the page. And if the user tries to enter something that’s not a number, there’s no explicit feedback about what they’re doing wrong.
 
-### Hint text
+### Prefixes and suffixes
```

Comment:
Do we want to include an example with both a prefix and a suffix in this section?

Reply:
Could you explain what you mean? We have an example with a prefix and a suffix, and then examples with one or the other in the following sections.

Reply:
🙈